src/SprykerEco/Zed/UnzerGui/Communication/Form/Constraint/UnzerCredentialsConstraintValidator.php

```diff
@@ -15,7 +15,7 @@
 class UnzerCredentialsConstraintValidator extends ConstraintValidator
 {
     /**
-     * @param \Generated\Shared\Transfer\UnzerCredentialsTransfer $value
+     * @param \Generated\Shared\Transfer\UnzerCredentialsTransfer|mixed $value
```

Comment:
this change shopuld make at least CI failing
that means $value is potentially not an instance of `UnzerCredentialsTransfer`
but the following call does not accept anything except transfer I mentioned
> $unzerCredentialsResponseTransfer = $constraint->getUnzerFacade()->validateUnzerCredentials($value);

Reply:
changes should be reverted

Reply:
Inherits from symfony and symfony requires mixed. Can we check for the type in the code? Validation can fail if the type is incorrect.

Reply:
Updated code, to check constraints.